### PR TITLE
Use CommonUxComponents controls in SharedUx and AccessibilityInsights

### PR DESCRIPTION
**Pull request by jalkire**
#### Describe the change
Previously, MessageDialog was duplicated into the new CommonUxComponents project. This PR follows that up by removing the original MessageDialog from SharedUx and updating all references accordingly. There should be no user impact. It also fixes the dialog's button styling.

FabricIconControl and DualFabricIconControl were duplicated into CommonUxComponents. This PR updates all usage of these controls in SharedUx and AccessibilityInsights. However, the original controls still reside in DesktopUI as well. They will be removed when (if) the highlighters are moved into SharedUx/out of the runtime.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@ public MessageDialog()
         {
             var dlg = new MessageDialog() { Message = message };
             dlg.Owner = Application.Current.MainWindow;
+            dlg.Resources = Application.Current.MainWindow.Resources;
```
Comment (neocln) [inline src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs]: Is there any implication from it? such as a certain resource is expected in App level resources?

Comment (jalkire) [reply src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs]: This is here to give the dialog access to the resources in Styles.xaml, specifically BtnBlue. In general, it gives the dialog access to whatever resources MainWindow has.

Comment (neocln) [reply src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs]: If so, don't we need to consider splitting Styles.xaml into two parts(common and AI-Win specific)?

Comment (jalkire) [reply src/AccessibilityInsights.CommonUxComponents/Dialogs/MessageDialog.xaml.cs]: Consider, yes. I would call that out of scope for this PR. There are more implications to deal with. Currently, SharedUx still owns theme, including styles, brushes, and font size. I'm not positive about separating those things out.

File: src/AccessibilityInsights.CommonUxComponents/Controls/FabricIconControl.xaml.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public FabricIconControl()
         /// Name of icon to be drawn
         /// </summary>
         #region GlyphName (Dependency Property)
-
```
Comment (DaveTryon) [inline src/AccessibilityInsights.CommonUxComponents/Controls/FabricIconControl.xaml.cs]: Remove line 33? (extra line and trailing spaces)

**Pull request merged**